ipam/allocator.go

```diff
@@ -264,6 +251,33 @@ func (alloc *Allocator) free(ident string) error {
 	return <-errChan
 }
 
+// Free (Sync) - release single IP address for container
+func (alloc *Allocator) Free(ident string, addr address.Address) error {
+	errChan := make(chan error)
+	alloc.actionChan <- func() {
+		addrs := alloc.owned[ident]
+		for i := range addrs {
+			if addrs[i] == addr {
```

Comment:
The code refers to some variant of `addrs[i]` three times, so to give one of them a different name seems more confusing to me.  I presume the compiler can figure out it's the same thing either way.